examples/Atmosphere.js

```diff
@@ -27,7 +27,6 @@ requirejs(['./WorldWindShim',
         
         var atmosphereLayer = new WorldWind.AtmosphereLayer();
         var timeStamp = Date.now();
```

Comment:
The declaration and initialization of timeStamp seems out of place. Is there a better location?

Reply:
Yes, I think line 36 would be better. It would be grouped with the other variables declared for the simulation component.